modules/web/src/app/wizard/step/provider-settings/provider/extended/openstack/default/template.html

```diff
@@ -41,10 +41,12 @@
                [formControlName]="Controls.Network"
                (changed)="onNetworkChange($event)"
                [label]="networksLabel"
+               [valueFormatter]="networkDisplayName.bind(this)"
                inputLabel="Select network..."
-               filterBy="name">
+               filterBy="name"
+               selectBy="id">
     <div *option="let network">
-      {{network.name}}
+      {{network.name || network.id}}
```

Comment:
We should display the ID in (ID: <network_id>) format here as well.
```suggestion
      {{network.name || network.id}}
```